apis/config/v1beta1/configuration_types.go

```diff
@@ -74,6 +74,10 @@ type WaitForPodsReady struct {
 	// is cancelled and requeued in the same cluster queue. Defaults to 5min.
 	// +optional
 	Timeout *metav1.Duration `json:"timeout,omitempty"`
+
+	// BlockAdmission when true, cluster queue will block admissions for all subsequent jobs
+	// until the jobs reach the PodsReady=true condition. It defaults to false.
+	BlockAdmission bool `json:"blockAdmission,omitempty"`
```

Comment:
The documentation of this field is contradicting the comment for `enable`.

I think we can say that enable is about tracking the PodsReady condition, but not about blocking.

Then this field is about blocking, but it should be true by default, because that's the existing behavior.
To implement true as default, you either need to make it a pointer `*bool` and check for nil.

Or you can change the name of the field to something like `admissionMode` with values `Sequential` or `Parallel`

Reply:
Sure, I will use blockAdmission as *bool and set the default as true.

Reply:
feel free to mark as resolved (along with other addressed comments)